src/components/App/App.jsx

```diff
@@ -141,36 +176,55 @@ function App() {
       })
       .finally(setIsLoading(false));
```

Comment:
` .finally(setIsLoading(false));`


In `then`, `catch`, `finally` you need to pass only the **reference** to the function (not a function call) otherwise the function is called immediately without waiting for the response from the server. You can pass an **unnamed** arrow function as well. 


```js
      .finally(() => setIsLoading(false));


```